docs/user-guide/Authentication.md

```diff
@@ -30,4 +32,9 @@ Note: in the above example, there are two different applications.  This is only
 
 **Token values in logs**
 
-RESTler has logic to prevent token values from being written to the network logs.  It is recommended to check the RESTler network logs and make sure that the token values are, indeed,  successfully omitted from the logs. 
\ No newline at end of file
+RESTler has logic to prevent token values from being written to the network logs.  It is recommended to check the RESTler network logs and make sure that the token values are, indeed,  successfully omitted from the logs. 
+                
+**Certificate based authentication**
+                
+Certificate and corresponding keys can be used as an authentication machanism. To use this add a key value pair in the settings.json file e.g. 
+                "client_certificate_key_path":"C:\\Users\\<folder></folder>\\privateKey.txt"
```

Comment:
>client_certificate_key_path [](http://example.com/codeflow?start=17&length=27)

This setting should be added to SettingsFile.md, right below client_certificate_path.  

It should say something like:

Path to the keyfile that is needed to load the certificate.

If both the keyfile and certificate path are valid, RESTler will attempt to use it during the SSL handshake.